<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationWindow</class>
 <widget class="QMainWindow" name="ConfigurationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configurations</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Player</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="useBuiltinPlayer">
          <property name="text">
           <string>Use Builtin Player</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="useExternalPlayer">
          <property name="text">
           <string>Use External Player</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>External Player Path</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="externalPlayerPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="browseExternalPlayer">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>External Player Commandline Arguments</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="externalPlayerArguments"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>External Player Working Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="externalPlayerWorkingDir"/>
          </item>
          <item>
           <widget class="QPushButton" name="browseExternalPlayerWorkingDir">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="reverseProxyTab">
       <attribute name="title">
        <string>Reverse Proxy</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Listen Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="reverseProxyListenPort">
          <property name="minimum">
           <number>1024</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Bind Network Interface</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="reverseProxyBindNetworkInterface">
          <item>
           <property name="text">
            <string>-- Default --</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="reverseProxyAutoRedirect">
          <property name="text">
           <string>Auto Redirect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="reverseProxyRedirect">
          <property name="text">
           <string>Redirect</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Proxy Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="reverseProxyProxyType">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Http</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Socks5</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Proxy Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="reverseProxyProxyAddress"/>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
